packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -474,6 +475,33 @@ export default class ConversationService {
     });
   }
 
+  public async addUser(conversationId: string, userId: string): Promise<string[]>;
+  public async addUser(conversationId: string, userIds: string[]): Promise<string[]>;
+  public async addUser(conversationId: string, userIds: string | string[]): Promise<string[]> {
+    const ids = typeof userIds === 'string' ? [userIds] : userIds;
+    return this.apiClient.conversation.api
+      .postMembers(conversationId, ids)
+      .then(() => ids)
+      .catch((error: AxiosError) => {
+        if (error && error.response) {
+          throw BackendErrorMapper.map(error.response.data as BackendError);
+        }
+        throw error;
+      });
+  }
+
+  public async removeUser(conversationId: string, userId: string): Promise<string> {
+    return this.apiClient.conversation.api
+      .deleteMember(conversationId, userId)
+      .then(() => userId)
+      .catch((error: AxiosError) => {
+        if (error && error.response) {
+          throw BackendErrorMapper.map(error.response.data as BackendError);
+        }
+        throw error;
+      });
```

Comment:
Function is `async`, too

Reply:
async/await is not always the most readable option: https://vote4code.com/5646874153320448/asyncawait